### ISSUE TITLE
Session expired handling / bug

### ISSUE THREAD

**Issue by StrayCat1** [cedardevs/onestop#1318]
### Summary
<table>
  <tr>
    <td><b>As a</b></td>
    <td>OneStop user</td>
  </tr>
  <tr>
    <td><b>I want to be able to</b></td>
    <td>
       see when I am logged in
    </td>
  </tr>
  <tr>
    <td><b>So that I can</b></td>
    <td>not be confused when it doesnt show my dashboard</td>
  </tr>
</table>

Currently if you login, and then your session expires before you logout, you will have a weird user experience. 

Basically Gateway thinks you are logged in but the User app does not. Which to me indicates that the access_token that the User app is using to talk to login.gov has expired but the session gateway established with the client has not. 

Will need to gather more logs to really know what's happening here. For now it remains kind of a big problem if you dont know how to manually logout (/logout) with Gateway.

**Comment by mcquinne** [comment]
Workarounds for local development in the meantime:

- Delete/recreate gateway pod to eliminate sessions
- Visit `localhost/logout` via k8s ingress or `localhost:8888/logout` when running via webpack dev server

**Comment by StrayCat1** [comment]
This issue was resolved temporarily by a workaround described in this issue
https://github.com/spring-cloud/spring-cloud-security/issues/175

Specifically this comment- 
https://github.com/spring-cloud/spring-cloud-security/issues/175#issuecomment-557135243

However it looks like it was resolved in a new spring cloud security release. Will need to follow up to see what the latest method for this is. 

With this temp fix in place, there is no 500 error from User. Gateway never sends the bad access_token that was causing it.
